### PR TITLE
Tag ApproxFun v0.5.0 [https://github.com/JuliaApproximation/ApproxFun…

### PR DESCRIPTION
**Pull request by dlfivefifty**
….jl]

Drops 0.4 support and now depends on RecipesBase instead of Plots.

### CODE REVIEW

File: ApproxFun/versions/0.5.0/requires [hunk 1]
```diff
@@ -0,0 +1,11 @@
+julia 0.5
+FastGaussQuadrature 0.1
+FastTransforms 0.0.7
+RecipesBase 0.1
+Compat 0.12
```
Comment (tkelman) [inline ApproxFun/versions/0.5.0/requires]: are you relying on https://github.com/JuliaLang/Compat.jl/pull/302 ?

**Comment by dlfivefifty** [comment]
No. On second thought I shouldn't need Compat at all since this only supports 0.5, but as soon as I update for 0.6 it will be needed again.

**Comment by dlfivefifty** [comment]
Can this be merged?

**Comment by tkelman** [comment]
I would remove the bounds on the Compat dependency if you aren't using it at all right now.

**Comment by dlfivefifty** [comment]
I don't know how to retag without creating a new pull request, so I'll do that.

**Pull request closed without merging**